recipes/code-aster/LICENSE.txt

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2015-2022, conda-forge
```

Comment:
This is supposed to be the license of the package (which is not owned by conda-forge). Often, the license is included in the source tarball, so you usually don't need to add it manually because conda-build looks for the license AFTER unpacking the source.

Reply:
I agree with that, I also prefer to use the one inside the package. In this case, the official archive is an "archive of archives" (the license file is inside a second layer of archive) so the magic to find the license inside will not work.

Reply:
I just tested it again : the exact error message is "conda.CondaError: Downloaded bytes did not match Content-Length". Of course, when downloaded manually, the hash is the same as provided in alternative link.

Reply:
I deleted this license file because it is not the correct type of license file and the copyright holder is listed as conda-forge.